test/models/group_offer_category_test.rb

```diff
@@ -1,15 +1,15 @@
 require 'test_helper'
 
 class GroupOfferCategoryTest < ActiveSupport::TestCase
-  def setup
-    @category = create :group_offer_category
-  end
-
   test 'group offer category is valid' do
-    assert @category.valid?
+    category = GroupOfferCategory.create!(category_state: 'active', category_name: 'bogus category')
```

Comment:
Couldn't that be in a `def setup`?

Reply:
The problem was that the tests failed on travis only. On travis it seemed that either dev setup didn't get executed or `@category` got lost. It only happened on travis. Thats why I changed it. Actually I didn't temper with this thing at all, just got the test back up running again.